olive/passes/onnx/onnxscript_fusion.py

```diff
@@ -33,7 +36,9 @@ def _run_for_config(
         model_ir = ir.from_proto(model_proto)
 
         # TODO(exporter team): Different fusions support different devices
-        ort_fusions.optimize_for_ort(model_ir)
+        model_ir, function_stats = ort_fusions.optimize_for_ort(model_ir)
```

Comment:
Can you change this as below? I think that will fix it for both release (0.2.3) and dev versions:
```py
output = ort_fusions.optimize_for_ort(model_ir)
if isinstance(output, tuple):
   model_ir, function_stats = output
else:
   model_ir = output
```

Reply:
Done